bot/seasons/evergreen/snakes/utils.py

```diff
@@ -532,8 +535,9 @@ def startup_event_check(reaction_: Reaction, user_: Member):
 
     async def cancel_game(self, user: Member):
         """Allow the game author to cancel the running game."""
-        if not user == self.author:
-            await self.channel.send(user.mention + " Only the author of the game can cancel it.", delete_after=10)
+        if not user == self.author and Roles.moderator not in [role.id for role in user.roles]:
+            await self.channel.send(user.mention + " Only the author of the game and server mods can cancel it.",
```

Comment:
In this case, the preferred indentation style in our repos would be:

```py
await self.channel.send(
    user.mention + " Only the author of the game and server mods can cancel it.",
    delete_after=10,
)
```

Like here: https://github.com/python-discord/seasonalbot/blob/abec4415d73ee5838c79e0a488d914589ae20ac2/bot/seasons/evergreen/snakes/utils.py#L556-L559

I also think that more recent additions to either bot or seasonalbot would use an `f-string` here, but that's not the prevalent style in this particular file. (The code here is quite old in PyDis terms.)